Funcky/Sequence.cs

```diff
@@ -1,14 +1,47 @@
+using System;
 using System.Collections.Generic;
 using System.Diagnostics.Contracts;
 using System.Linq;
 using Funcky.GenericConstraints;
+using Funcky.Monads;
 
 namespace Funcky
 {
     public static class Sequence
```

Comment:
Split the Sequence class into partials for the individual functions.

Reply:
646cebcdcf0b7f874b04d4cde924ae6a15eb60da